src/Content/RichContent.js

```diff
@@ -0,0 +1,21 @@
+import { createElement, Component } from 'react';
+import { string } from 'prop-types';
+
+export default class RichContent extends Component {
+    static propTypes = {
+        sanitizedRawHtml: string.isRequired
+    };
+
+    render() {
+        return (
+            this.props.children || (
+                <span
+                    className="peregrine-raw-html"
```

Comment:
We're not encouraging use of global CSS, so I don't think it's a good idea to have a generic class on here. Would prefer `className={this.props.className}`